cmd/edgex-secrets-config/README.md

```diff
@@ -0,0 +1,193 @@
+% edgex-secrets-config-proxy(1) User Manuals edgex-secrets-config-proxy(1)
+
+# NAME
+
+edgex-secrets-config-proxy – Configure EdgeX proxy service
+
+# SYNOPSIS
+
+**edgex-secrets-config proxy** SUBCOMMAND [OPTIONS]
+
+# DESCRIPTION
+
+Configures the EdgeX proxy service.
+
+This command is used to configure the TLS certificate for external connections, create authentication tokens for inbound proxy access, and other related utility functions.
+
+Proxy configuration commands (listed below) require access to the secret store master key in order to generate temporary secret store access credentials.
+
+# OPTIONS
+
+  * **--confdir** _/path/to/directory/with/configuration.toml_ (optional)
+
+    Points to directory containing a configuration.toml file.
+
+# SUBCOMMANDS
+
+  * **tls**
+
+    Configure inbound TLS certificate. This command will provision the TLS secrets into the secret store and re-deploy them to Kong. Requires additional arguments:
+
+    * **--in** _/path/to/certchain.crt_ (required)
```

Comment:
change to `--incert` to be consistent with `--inkey`

Reply:
@tonyespy Please weigh in.  Is was your preference to be consistent with openssl.

Reply:
The openssl x509 command uses `--in` for it's main input argument (if not specified stdin is used), and then uses qualified names (e.g. `--signkey infile`) for other input options. I'm not opposed to making this `--incert`, but if we do that, then I don't think it makes sense to also support stdin.

Reply:
Fine with me.  I will change to --incert then.